po4a

```diff
@@ -392,7 +392,7 @@ Do not change the POT and PO files, only the translation may be updated.
 
 Keeps the existing translation files even if the translation doesn't meet the threshold specified by --keep.
 This option does not create new translation files with few content,
```

Comment:
For some odd reason there are two whitespaces here.

Reply:
The only reason is my mistake, I guess. I'll fix it.

Reply:
I fail to see the two whitespaces, sorry. Feel free to submit another PR.

Reply:
You see them in the diff, but not anywhere else. Possibly zero-width spaces?